tests/web/rpc/v1alpha/test_trace_item_attribute_list.py

```diff
@@ -5,17 +5,17 @@
 import pytest
 from google.protobuf.timestamp_pb2 import Timestamp
 from sentry_protos.snuba.v1alpha.endpoint_tags_list_pb2 import (
-    TraceItemAttributesRequest,
+    TraceItemAttributesRequest as TraceItemAttributesRequestProto,
```

Comment:
is annoying to have to do default imports like this, can we just make sentry-protos name things blahproto by default?